Makefile

```diff
@@ -168,13 +168,27 @@ check-copyright:
 ###############################################################################
 # Release targets 
 ###############################################################################
+PREVIOUS_RELEASE=$(shell git describe --tags --abbrev=0)
```

Comment:
Does always work (e.g. with patch releases etc..)

Reply:
yeppers

Reply:
It won't work for the first release in a new stream - e.g. the first release vX.Y.0  from `release-vX.Y` after it was cut.

That's why the "cutting a minor release" section has this overridden explicitly.